config/settings.yml

```diff
@@ -247,7 +247,7 @@ aca:
       minimum_commission_statement_year: 2017
 
     transmit_scheduled_employers: true
-    employer_transmission_day_of_month: 26
+    employer_transmission_day_of_month: 27
```

Comment:
This change of date should have no issues code wise. It will transmit on the 27th according to when the event is triggered and the criteria is matched.